esp/esdllib/esdl_xpath_extensions_libxml.cpp

```diff
@@ -806,6 +806,138 @@ static void getTxSummary(xmlXPathParserContextPtr ctxt, int nargs)
     xmlXPathReturnString(ctxt, xmlStrdup((const xmlChar*)output.str()));
 }
 
+static void getMaskValueBehavior(xmlXPathParserContextPtr ctxt, int nargs)
+{
+    IEsdlScriptContext *scriptContext = queryEsdlScriptContext(ctxt);
+    if (!scriptContext)
+    {
+        xmlXPathSetError((ctxt), XPATH_INVALID_CTXT);
+        return;
+    }
+    Owned<IDataMaskingProfileContext> masker(scriptContext->getMasker());
+
+    if (nargs < 1 || nargs > 2)
+    {
+        xmlXPathSetArityError(ctxt);
+        return;
+    }
+
+    StringBuffer valueType, maskStyle;
+    xmlChar* tmp;
+    if (2 <= nargs)
```

Comment:
nitpick: Inconsistent order of variable vs. literal in comparison. Maybe keep variable first unless a strict equality check?

Reply:
I agree it is harder to read (I initially thought the code was wrong).  Probably not worth changing for this PR though - possibly go back and revisit.

Reply:
https://track.hpccsystems.com/browse/HPCC-29156